assets/scss/sections/_about.scss

```diff
@@ -1,91 +1,154 @@
-  position: absolute;
-  bottom: calc(50% - 40px);
-  left: calc(50% - 30px);
-}
+    &-text {
+      color: var(--color-primary-500);
+      margin-block: var(--spacing-4);
+    }
+  }
 
-.red-dot {
-  width: 15px;
-  height: 15px;
-  background-color: #bd1919;
-  border-radius: 50%;
-  position: absolute;
-  top: 20px;
-  left: 20px;
-}
+  &__item {
+    display: flex;
+    align-items: center;
+    gap: var(--spacing-1);
+    margin-block-end: var(--spacing-1);
+  }
+  &__item-icon {
+    color: var(--color-primary-200);
+    font-size: 2rem;
+    flex-shrink: 0;
+  }
+  &__ring-container {
+    position: absolute;
+    bottom: calc(50% - var(--spacing-5));
+    left: calc(50% - var(--spacing-4));
 
-.ringring {
-  border: 3px solid #bd1919;
-  border-radius: 30px;
-  height: 25px;
-  width: 25px;
-  position: absolute;
-  left: 15px;
-  top: 15px;
-  animation: pulsate 1s ease-out;
-  animation-iteration-count: infinite;
-  opacity: 0;
-}
-@keyframes pulsate {
-  0% {
-    -webkit-transform: scale(0.1, 0.1);
-    opacity: 0;
+    &__red-dot {
+      width: var(--spacing-2);
+      height: var(--spacing-2);
+      background-color: #bd1919;
+      border-radius: 50%;
+      position: absolute;
+      top: 20px;
+      left: 20px;
+    }
+    &__ringring {
+      border: 3px solid #bd1919;
+      border-radius: var(--spacing-4);
+      height: var(--spacing-3);
+      width: var(--spacing-3);
+      position: absolute;
+      left: var(--spacing-2);
+      top: var(--spacing-2);
+      -webkit-animation: pulsate 1s ease-out;
+      animation: pulsate 1s ease-out;
+      -webkit-animation-iteration-count: infinite;
+      animation-iteration-count: infinite;
+      opacity: 0;
+    }
+  }
+  @-webkit-keyframes pulsate {
+    0% {
+      transform: scale(0.1, 0.1);
+      opacity: 0;
+    }
+    50% {
+      opacity: 1;
+    }
+    100% {
+      transform: scale(1.2, 1.2);
+      opacity: 0;
+    }
   }
-  50% {
-    opacity: 1;
+  @keyframes pulsate {
+    0% {
+      transform: scale(0.1, 0.1);
+      opacity: 0;
+    }
+    50% {
+      opacity: 1;
+    }
+    100% {
+      transform: scale(1.2, 1.2);
+      opacity: 0;
+    }
   }
-  100% {
-    -webkit-transform: scale(1.2, 1.2);
+  &__ring {
+    border: 3px solid #bd1919;
+    border-radius: var(--spacing-4);
+    height: var(--spacing-3);
+    width: var(--spacing-3);
+    position: absolute;
+    left: var(--spacing-2);
+    top: var(--spacing-2);
+    -webkit-animation: pulsate 1s ease-out;
+    animation: pulsate 1s ease-out;
+    -webkit-animation-iteration-count: infinite;
+    animation-iteration-count: infinite;
     opacity: 0;
   }
+
+  @media (min-width: 768px) {
+    &__container {
+      gap: 40px;
+    }
+
+    &__tab {
+      &-list {
+        display: grid;
+        grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+      }
+      &-btn {
+        max-width: -webkit-max-content;
+        max-width: -moz-max-content;
+        max-width: max-content;
```

Comment:
Why do you keep using the prefixes? As we discussed it's not your job, it's autoprefixer's

Reply:
I installed autoprefixer in vscode and that extension added prefixes.